panel/io/cache.py

```diff
@@ -138,13 +138,58 @@ def _pandas_hash(obj):
         # it contains unhashable objects.
         return b"%s" % pickle.dumps(obj, pickle.HIGHEST_PROTOCOL)
 
+def _polars_combine_hash_expr(columns):
```

Comment:
The `hash().sum().sum_with_columns()` was not sensitive to ordering as it would just convert all values to a number and sum it.